src/smc-webapp/editor_chat.cjsx

```diff
@@ -287,6 +297,15 @@ class ChatActions extends Actions
         w.document.write('</body></html>')
 
         w.addEventListener "unload", () =>
+            video = (@store.get('video')?.toJS()) ? {}
+            users = video.users
+            # deletes the user from the user list
+            index = users.indexOf(account_id)
+            if index != -1
+                users.splice(index, 1)
+                video.users = users
```

Comment:
This line `video.users = users` doesn't do anything, since `video.users` and `users` are just references to the same object.